### ISSUE TITLE
Subsequent runs of terraform fail after initial creation of single-node type cluster in aws_redshift_cluster resource.

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#554]
_This issue was originally opened by @swettk as hashicorp/terraform#12220. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
Terraform v0.8.7

### Affected Resource(s)
aws_redshift_cluster

### Terraform Configuration Files
```hcl
resource "aws_redshift_cluster" "default-single" {
  cluster_identifier = "${lower(var.tag_environment_name)}-${lower(var.tag_project_name)}"
  database_name = "${var.db_name}"
  master_username = "${var.master_user}"
  master_password = "${var.master_password}"
  node_type = "${var.node_type}"
  cluster_type = "single-node"
  enhanced_vpc_routing = "true"
  vpc_security_group_ids = [ "${aws_security_group.redshift_sg.id}" ]
  publicly_accessible = "false"
  cluster_subnet_group_name = "${aws_redshift_subnet_group.redshift_subnet_group.name}"
}
```

### Expected Behavior
```terraform apply``` on subsequent runs complete without error.

### Actual Behavior
1 error(s) occurred:

* aws_redshift_cluster.default-single: [WARN] Error modifying Redshift Cluster (development-snowplow): InvalidParameterCombination: Number of nodes for cluster type multi-node must be greater than or equal to 2
	status code: 400, request id: 8f6c2575-fa22-11e6-be91-f5ed3118d3ad

### Steps to Reproduce
1. `terraform apply`
1. `terraform apply` again

### Important Factoids
Nothing odd I hope.

**Comment by bflad** [comment]
The fix for this should be contained in #3127 and will release with v1.18.0 of the AWS provider in two days.

**Comment by bflad** [comment]
This has been released in [version 1.18.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1180-may-10-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
